client/src/components/CreatePost/StyledPostAs.js

```diff
@@ -166,6 +166,10 @@ const StyledSvgIcon = styled(SvgIcon)`
   pointer-events: none;
 `;
 
+const OptionButtonWrapper = styled.div`
+overflow: auto;
```

Comment:
this needs to be indented too.

Reply:
> this needs to be indented too.

fixed , sorry my bad.. I had to check this before I did PR